pkg/log/zap/zap_test.go

```diff
@@ -423,6 +423,17 @@ var _ = Describe("Zap log level flag options setup", func() {
 			Expect(out.StacktraceLevel.Enabled(zapcore.ErrorLevel)).To(BeTrue())
 		})
 
+		It("Should output stacktrace at panic level, with development mode set to true.", func() {
+			args := []string{"--zap-stacktrace-level=panic", "--zap-devel=true"}
+			fromFlags.BindFlags(&fs)
+			err := fs.Parse(args)
+			Expect(err).ToNot(HaveOccurred())
+			out := Options{}
+			UseFlagOptions(&fromFlags)(&out)
+
+			Expect(out.StacktraceLevel.Enabled(zapcore.PanicLevel)).To(BeTrue())
```

Comment:
Isn't the more important test that its disabled on the next level, i.E. `error`?

Reply:
Added disable check for `info` and `error`.